lib/project_types/extension/models/specification_handlers/default.rb

```diff
@@ -103,6 +103,10 @@ def build_resource_url(shop)
           raise NotImplementedError
         end
 
+        def server_config_file
+          "shopifile.yml"
```

Comment:
This is temporary, as the name has not been agreed upon yet. 
Defining this in the `specification_handler` will allow other specification types to define what their config file is.